.travis.yml

```diff
@@ -14,6 +14,7 @@ cache:
     - node_modules
 
 script:
+  - yarn build
```

Comment:
I would run this after the tests/linters

Reply:
Some of the tests rely on the bundle though? All of the tests in `playground` rely on it, and they are being ran as well. [Here](https://github.com/commercetools/merchant-center-application-kit/blob/master/playground/src/components/channels-list/channels-list.spec.js) for example.

If this runs without the bundle, you get errors like 

```
    Cannot find module '@commercetools-frontend/application-shell' from 'entry-point.spec.js'
```
We could split the test run in two I guess?

Reply:
Hmm what do you mean? Nothing should rely on the bundle...I'm confused

Reply:
The `playground` relies on the bundle right? So it stands to reason that any test in `playground`, requires the bundle to be built.